frontend/lib/ui/Icon/Icon.module.css

```diff
@@ -20,6 +26,42 @@
     }
   }
 
+  &.errors_and_warnings {
```

Comment:
we need a convention on classnames, personally dont like underscores but I cant explain why :)

Reply:
Changed underscores to dashes in a755932350ec4745bb3dd803a27c9ce33eaa3c62